sql/oracle_fdw.sql

```diff
@@ -8,7 +8,7 @@ SET client_min_messages = WARNING;
 CREATE EXTENSION oracle_fdw;
 
 -- TWO_TASK or ORACLE_HOME and ORACLE_SID must be set in the server's environment for this to work
-CREATE SERVER oracle FOREIGN DATA WRAPPER oracle_fdw OPTIONS (dbserver '', isolation_level 'read_committed', nchar 'true');
+CREATE SERVER oracle FOREIGN DATA WRAPPER oracle_fdw OPTIONS (dbserver '//192.168.1.26:1521/orcl', isolation_level 'read_committed', nchar 'true');
```

Comment:
I have left the server empty deliberately - I am using a local connection.
I could agree to 127.0.0.1, but not to 192.168.1.26